packages/core/chrome/core-chrome-browser-internal/src/ui/loading_indicator.tsx

```diff
@@ -76,7 +77,7 @@ export class LoadingIndicator extends React.Component<LoadingIndicatorProps, { v
       />
     ) : (
       <EuiIcon
-        type="logoElastic"
+        type={this.props.customLogo ?? 'logoElastic'}
```

Comment:
NIT/question: Are we going to only allow customers to choose an eui icon as their customer loading image?

Reply:
Very good catch, I am not sure how this logic got here 🤔 We shouldn't be customizing elastic spinner, but show plain `EuiLoadingSpinner` from line 72 if custom logo is set. Will fix!